core/templates/dev/head/services/utils.service.ts

```diff
@@ -36,6 +36,30 @@ export class UtilsService {
   isString(input: any): boolean {
     return (typeof input === 'string' || input instanceof String);
   }
+
+  isEquivalent(a: any, b: any): boolean {
```

Comment:
Thanks for this Ankita, this is useful :).

Reply:
You're welcome!